test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -90,7 +115,7 @@ describe('POST /group/:groupId/join', () => {
     });
   });
 
-  context('Accepting invitation to a party', () => {
+  context('Joining a party', () => {
     let user, invitedUser, party;
```

Comment:
Does this syntax pass lint? I'm pretty sure ES6 wants us to declare variables on new lines. I may be wrong.